src/components/docs/Buckets/Bucket/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
Not sure "bucket" makes sense here. I can't see the component and going to Jira doesn't help. Is this the card container?

Reply:
It's not a component but rather the base wrapper for the other components.... Do you think it'd be better to call it Base? It's in the Bucket folder so it would be implied that Base would be referring to Bucket